Clarified where to set the ENABLE_OPENBAGES variable

czoellick-icf opened the pull request:
## [DOC-XXXX](https://openedx.atlassian.net/browse/DOC-XXXX)

Clarified where to set the ENABLE_OPENBAGES variable in the configuration documentation set, section 4.7.3.

### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check/copy edit/dev edit): 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

edx-webhook commented:
Thanks for the pull request, @czoellick-icf! I've created [OSPR-1337](https://openedx.atlassian.net/browse/OSPR-1337) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



If you like, you can add yourself to the [AUTHORS](https://github.com/edx/edx-documentation/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

lamagnifica commented:
Thanks for this contribution, @czoellick-icf !
@nedbat this looks good to me from a doc team perspective. 👍

czoellick-icf commented:
I added another update to the theme activation documentation and fixed an issue from the CI tests. Definitely please let me know if there are additional steps that I need to take. I don't know enough about the reviewer process to route this to appropriate people.

nedbat commented:
@lamagnifica Take one last look, then merge when you are ready.

lamagnifica commented on en_us/install_operations/source/configuration/theming/activate_theme.rst:
```diff
@@ -12,25 +12,46 @@ you must update the configuration scripts to install your theme onto the server,
 as well. It is up to you where to install the theme on the server, but
 a good default location is ``/edx/app/themes/my-theme``.
 
+There are three core variables that must be updated to enable  enable a
```
duplicated "enable"

lamagnifica commented on en_us/install_operations/source/configuration/theming/activate_theme.rst:
```diff
@@ -12,25 +12,46 @@ you must update the configuration scripts to install your theme onto the server,
+There are three core variables that must be updated to enable  enable a
+comprehensive theme:
+
+- ``ENABLE_COMPREHENSIVE_THEMING`` - a boolean value that enables the
```
capitalize Boolean

lamagnifica commented on en_us/install_operations/source/configuration/theming/activate_theme.rst:
```diff
@@ -12,25 +12,46 @@ you must update the configuration scripts to install your theme onto the server,
+  COMPREHENSIVE_THEME_DIRS: ["/edx/app/themes", ],
+  DEFAULT_SITE_THEME: "my-theme"
+
+You will then need to restart the LMS in order for it to pick up these value
```
value^s

czoellick-icf commented:
@lamagnifica I fixed the items you found in your review.

lamagnifica commented:
Thank you very much, @czoellick-icf !
